LDM-592.tex

```diff
@@ -963,7 +963,7 @@ \section{Data Release Processing}
 The Validation Scientist analyzes a subset of the new data release by directly comparing it with the current Data Release.
 
 \item
-A statistical analysis is done with the combined catalogs to determine if the new data release is suitable for release.
+A statistical analysis is done with the catalogs derived from both data releases to determine if the new data release is suitable for release.
```

Comment:
I'm happy with this adjustment; it's consistent with the spirit of what I had in mind.